### PR TITLE
Update CONTRIBUTING.md to include building JS

### PR DESCRIPTION
**Pull request by sunboyy**
This pull request updates CONTRIBUTING.md in the Building Gitea section. I added information about building JavaScript files so that new contributors can setup the project more easily.

### CODE REVIEW

**Review by sapk — APPROVED** [review]

**Review by axifive — APPROVED** [review]

**Review by zeripath — COMMENTED** [review]
I think this is out-of-date make build will generate the css and js now.

**Comment by lunny** [comment]
You can still use `make generate build` but `generate` could be ignored.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -114,10 +114,10 @@ included in the next released version.
 ## Building Gitea
 
 Generally, the go build tools are installed as-needed in the `Makefile`.
-An exception are the tools to build the CSS and images.
+An exception are the tools to build the CSS, JS and images.
 
-- To build CSS: Install [Node.js](https://nodejs.org/en/download/package-manager) at version 8.0 or above
-  with `npm` and then run `npm install` and `make css`.
+- To build CSS and JS: Install [Node.js](https://nodejs.org/en/download/package-manager) at version 8.0 or above
```
Comment (techknowlogick) [inline CONTRIBUTING.md]:
```suggestion
- To build CSS and JS: Install [Node.js](https://nodejs.org/en/download/package-manager) at version 10.0 or above
```

**Review by techknowlogick — DISMISSED** [review]

**Review by 6543 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (sunboyy) [reply CONTRIBUTING.md]: Fixed

**Comment by 6543** [comment]
ping lgtm

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -114,10 +114,10 @@ included in the next released version.
 ## Building Gitea
 
 Generally, the go build tools are installed as-needed in the `Makefile`.
-An exception are the tools to build the CSS and images.
+An exception are the tools to build the CSS, JS and images.
 
-- To build CSS: Install [Node.js](https://nodejs.org/en/download/package-manager) at version 8.0 or above
-  with `npm` and then run `npm install` and `make css`.
+- To build CSS and JS: Install [Node.js](https://nodejs.org/en/download/package-manager) at version 10.0 or above
+  with `npm` and then run `npm install`, `make css` and `make js`.
```
Comment (bagasme) [inline CONTRIBUTING.md:120]: `make js` what's means?

Comment (techknowlogick) [reply CONTRIBUTING.md:120]: `make js` is a command that generates the javascript for the application

**Pull request merged**